db/migrate/20160915144154_add_color_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddColorToTags < ActiveRecord::Migration
+  def change
+    add_column :tags, :color, :string, default: "#f50"
```

Comment:
So we could also have done it with CSS, but in this way it is default and it is already there, so we don't have to check if it is defined, because if we forget it, then the tags would have no background color. It is a default that makes sure we don't have to define it and that there is a background color per se.